### PR TITLE
feat: [sc-15958] Implement employability.md in api documentation

### PR DESCRIPTION
**Pull request by AStoerkel1**
Story details: https://app.shortcut.com/clarkcan/story/15958
image

### CODE REVIEW

**Review by pmanal1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

File: docs/competency-api/employability.md [hunk 1]
```diff
@@ -4,4 +4,51 @@ title: Employability
...
 parent: Competency API
 ---
-# Employability
\ No newline at end of file
+# Employability
+
+## Update Employability
+
+### Description
+
```
Comment (Cwagne17) [inline docs/competency-api/employability.md:13]: Needs a description

File: docs/competency-api/employability.md [hunk 2]
```diff
@@ -4,4 +4,51 @@ title: Employability
...
+#### patch:
+```http
+https://api.competency-constructor.clark.center/competencies/:competencyId/employability
+```
+
+### Headers
+```json
+{
+    Bearer: "Bearer Example.bearer.token"
```
Comment (Cwagne17) [inline docs/competency-api/employability.md]: Header is called authorization

File: docs/competency-api/employability.md [hunk 3]
```diff
@@ -4,4 +4,51 @@ title: Employability
...
+|:----:|:-----:|:----:|-----|
+| details | true | string | Any performance factors which are important to the competency and not otherwise specified |
+| userId | true | Object Id: string | The userId matching the Bearer token |
+
+<details closed markdown="block">
+  <summary>
+    Example
+  </summary>
+```json
+{
```
Comment (Cwagne17) [inline docs/competency-api/employability.md:39]: Curl request example

**Review by Cwagne17 — CHANGES_REQUESTED** [review]

File: docs/competency-api/employability.md (hunk 1 above)
Comment (JosueMolinaMorales) [reply docs/competency-api/employability.md:13]: Yeah a short description of what the route does and when it can be used wouldnt hurt.

**Review by JosueMolinaMorales — COMMENTED** [review]
Chris's comments

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**